docs/source/tools.rst

```diff
@@ -12,5 +12,20 @@ produced by the pipeline.
     repository <https://github.com/microbial-pangenomes-lab/gwas_template/issues>`__.
 
 * ``unitig-counter``: `10.1371/journal.pgen.1007758 <https://doi.org/10.1371/journal.pgen.1007758>`__ and `10.1128/mbio.01344-20 <https://doi.org/10.1128/mbio.01344-20>`__
-
-**Etcetera**
+* ``mlst``: `Seemann T, mlst <https://github.com/tseemann/mlst>`__ and `https://doi.org/10.12688/wellcomeopenres.14826.1 <https://wellcomeopenresearch.org/articles/3-124/v1>`__
+* ``poppunk``: `10.1101/gr.241455.118 <https://genome.cshlp.org/content/29/2/304>`__
+* ``abritamr``: `10.1038/s41467-022-35713-4 <https://doi.org/10.1038/s41467-022-35713-4>`__
+* ``mash``: `10.1186/s13059-016-0997-x <https://doi.org/10.1186/s13059-016-0997-x>`__
+* ``panaroo``: `10.1186/s13059-020-02090-4 <https://doi.org/10.1186/s13059-020-02090-4>`__
+* ``fasttree``: `10.1371/journal.pone.0009490 <https://doi.org/10.1371/journal.pone.0009490>`__
+* ``snp-sites``: `10.1099/mgen.0.000056 <https://doi.org/10.1099/mgen.0.000056>`__
+* ``bcftools``: `10.1093/gigascience/giab008 <https://doi.org/10.1093/gigascience/giab008>`__
+* ``limix``: `10.1101/003905 <https://doi.org/10.1101/003905>`__
+* ``ALBI``: `10.1016/j.ajhg.2016.04.016  <https://doi.org/10.1016/j.ajhg.2016.04.016>`__
+* ``sequence_unet``: `10.1186/s13059-023-02948-3 <https://doi.org/10.1186/s13059-023-02948-3>`__
+* ``pyseer``: `10.1093/bioinformatics/bty539 <https://doi.org/10.1093/bioinformatics/bty539>`__ and `10.1101/852426 <https://doi.org/10.1101/852426>`__
+* ``bwa``: `arXiv:1303.3997 <https://arxiv.org/abs/1303.3997>`__
+* ``bedtools``: `10.1093/bioinformatics/btq033 <https://doi.org/10.1093/bioinformatics/btq033>`__ and `10.1093/bioinformatics/btr539 <https://doi.org/10.1093/bioinformatics/btr539>`__
+* ``eggnog-mapper``: `10.1093/molbev/msx148 <https://doi.org/10.1093/molbev/msx148>`__
+* ``panfeed``: `10.1099/mgen.0.001129 <https://doi.org/10.1099/mgen.0.001129>`__
+* ``rhierbaps``: `10.12688/wellcomeopenres.14694.1 <https://wellcomeopenresearch.org/articles/3-93/v1>`__ and `10.1093/molbev/mst028 <https://doi.org/10.1093/molbev/mst028>`__
```

Comment:
Do we use this tool? I don't think so?

Reply:
Ah, no. The env was still there and I added it by error. I'll fix it now